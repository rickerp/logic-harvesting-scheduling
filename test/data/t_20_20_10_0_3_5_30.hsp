30
3
8 5 5 4 3 8 6 8 5 7 4 1 10 5 6 4 10 9 3 2 4 10 2 2 4 6 3 6 5 8
7 2 3 6 9 10 16 19
6 1 5 13 14 15 21
6 1 4 6 7 9 10
3 3 5 7
3 2 4 8
4 1 3 7 9
8 3 4 6 8 9 11 17 26
8 5 7 9 11 12 16 22 25
10 1 3 6 7 8 10 11 16 17 25
7 1 3 9 11 16 19 30
9 7 8 9 10 12 17 25 26 30
6 8 11 13 17 25 28
9 2 12 14 16 17 22 25 26 28
7 2 13 15 17 22 25 28
3 2 14 18
11 1 8 9 10 13 17 19 25 26 28 30
16 7 9 11 12 13 14 16 18 19 20 22 25 26 28 29 30
5 15 17 19 20 21
11 1 10 16 17 18 20 22 25 26 28 30
6 17 18 19 21 22 23
6 2 18 20 22 23 29
10 8 13 14 17 19 20 21 23 25 26
4 20 21 22 24
3 23 26 29
13 8 9 11 12 13 14 16 17 19 22 26 28 30
10 7 11 13 16 17 19 22 24 25 27
4 26 28 29 30
10 12 13 14 16 17 19 25 27 29 30
6 17 21 24 27 28 30
9 10 11 16 17 19 25 27 28 29
3 1 2 4 5 3 2 2 5 5 1 1 5 4 3 2 3 3 3 3 3 2 2 1 2 2 1 5 5 5
1 5 2 3 3 2 2 1 5 3 3 2 3 4 2 1 2 4 5 5 4 2 3 5 4 5 1 4 4 1
1 5 2 4 4 1 2 2 2 1 5 4 5 4 2 1 4 1 5 5 5 5 3 2 4 2 3 1 5 2
0
