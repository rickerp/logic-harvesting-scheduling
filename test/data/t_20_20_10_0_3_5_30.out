86
8 4 6 8 10 13 15 20 29
6 2 9 18 23 26 28
8 3 5 11 14 19 21 24 27
0
